std/numeric.d

```diff
@@ -2987,64 +2987,70 @@ if (isIntegral!T)
 // This overload is for non-builtin numerical types like BigInt or
 // user-defined types.
 /// ditto
-T gcd(T)(T a, T b)
+auto gcd(T)(T a, T b)
```

Comment:
I dislike `auto` as return type - it requires the user to look at the source to infer it; for the compiler, it means it always has to analyze the function to infer it too (but for templates it doesn't matter that much).